src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -520,7 +520,8 @@ private void AddTupleTypeName(INamedTypeSymbol symbol)
                 }
 
                 VisitFieldType(element);
-                if (!element.IsImplicitlyDeclared)
+                if (!element.IsImplicitlyDeclared &&
+                    !(symbol.IsGenericType && symbol.IsDefinition)) // C# tuples are generic
```

Comment:
Are we checking if this is a default field rather than a named field? If so, is there a more explicit way to do that check that is language-independent? Perhaps `element.CorrespondingTupleField != null`.

Reply:
I've tried to think of different approaches, and tried a few, but this is really tricky.
We don't want to display the `Item1` in `VT'2` (which is `!IsImplicitlyDeclared`, but has a `CorrespondingTupleField`).
Also checks are whether fields are default are not don't work, as `(int Item1, int).Item1` is a default field even though explicitly named.